core/services/nmea_injector/settings.py

```diff
@@ -0,0 +1,42 @@
+from typing import Any, Dict
+
+import pykson  # type: ignore
+from commonwealth.settings import settings
+
+
+class NmeaInjectorSettingsSpecV1(pykson.JsonObject):
+    kind = pykson.StringField()
+    port = pykson.IntegerField()
+    component_id = pykson.IntegerField()
+
+    @staticmethod
+    def from_spec(spec: "NmeaInjectorSpec") -> "NmeaInjectorSettingsSpecV1":  # type: ignore
```

Comment:
This should be `NMEASocket` when reading `async def add_sock(self, sock: NMEASocket) -> None:`

![image](https://user-images.githubusercontent.com/1215497/163674777-ab02fba5-f988-4c84-a02d-b3e624faf8fd.png)